azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -1308,27 +1310,22 @@ public String handle(ResultSet rs) throws SQLException {
     ResultSetHandler<List<Pair<ExecutionReference, ExecutableFlow>>> {
     // Select queued unassigned flows
     private static String FETCH_QUEUED_EXECUTABLE_FLOW =
-      "SELECT ex.exec_id exec_id, ex.enc_type enc_type, ex.flow_data flow_data, "
-        + " ax.update_time axUpdateTime FROM execution_flows ex"
-        + " INNER JOIN"
-        + " active_executing_flows ax ON ex.exec_id = ax.exec_id"
-        + " Where ex.executor_id is NULL";
+        "SELECT exec_id, enc_type, flow_data FROM execution_flows"
+            + " Where executor_id is NULL AND status = ?";
```

Comment:
Have you considered adding the status here instead of having the caller passing in a value? 
I think it will be easier to read and less likely for callers to pass in the wrong value. There is no need for different callers to pass in a different value. Right?